lib/fs.js

```diff
@@ -1122,60 +1117,41 @@ function rmdir(path, options, callback) {
     options = undefined;
   }
 
+  if (options?.recursive !== undefined) {
```

Comment:
Nit: would be nice to add a comment here. Either explaining why is this here (i.e. reference to DEP0147), or a TODO about removal in future major releases.